karpenter/metadata.csv

```diff
@@ -34,6 +39,8 @@ karpenter.cloudprovider.errors.count,count,,error,,The count of errors returned
 karpenter.cloudprovider.instance.type.cpu_cores,gauge,,core,,VCPUs cores for a given instance type,0,karpenter,,
 karpenter.cloudprovider.instance.type.memory_bytes,gauge,,byte,,"Memory, in bytes, for a given instance type",0,karpenter,,
 karpenter.cloudprovider.instance.type.price_estimate,gauge,,,,Estimated hourly price used when making informed decisions on node cost calculation. This is updated once on startup and then every 12 hours,0,karpenter,,
+karpenter.cluster_state.node_count,gauge,,node,,Current count of nodes in cluster state,0,karpenter,,
+karpenter.cluster_state.synced,gauge,,,,Returns 1 if cluster state is synced and 0 otherwise. Synced checks that nodeclaims and nodes that are stored in the APIServer have the same representation as Karpenter's cluster state,0,karpenter,,
```

Comment:
```suggestion
karpenter.cluster_state.node_count,gauge,,node,,Current count of nodes in cluster state.,0,karpenter,,
karpenter.cluster_state.synced,gauge,,,,Returns 1 if cluster state is synced and 0 otherwise. Synced checks that nodeclaims and nodes that are stored in the APIServer have the same representation as Karpenter's cluster state,0,karpenter,,
```